examples/slate-demo-explicit2.ipynb

```diff
@@ -0,0 +1,307 @@
+    "    \n",
+    "    fireflyDataHM = [\n",
+    "                {'dataType': 'fireflyHeatmap'},\n",
+    "                {'dataType': 'fireflyHeatmap'},\n",
+    "                {'dataType': 'fireflyHeatmap'}]\n",
+    "    fc.show_plot(group_id=viewer_id, data=dataHM, fireflyData=fireflyDataHM )    \n",
+    "    \n",
+    "# Add third chart - histogram  (plot.ly direct plot) \n",
+    "# in cell 2, 2, 2, 3\n",
+    "viewer_id =  'histContainer'\n",
+    "r = fc.add_cell(2, 2, 2, 3, 'xyPlots', viewer_id)\n",
+    "\n",
+    "if r['success']:\n",
+    "    dataH = [\n",
+    "        {'name': 'dec-0.02', 'marker': {'color': 'rgba(153, 51, 153, 0.8)'}},\n",
+    "        {'name': 'dec+0.02', 'marker': {'color': 'rgba(102,153,0, 0.7)'}}\n",
+    "    ]\n",
+    "    \n",
+    "    fireflyDataH = [\n",
+    "        {\n",
+    "            'dataType': 'fireflyHistogram',\n",
+    "            'tbl_id': 'wiseCatTbl',\n",
+    "            'options': {\n",
+    "                'algorithm': 'fixedSizeBins',\n",
+    "                'fixedBinSizeSelection': 'numBins',\n",
+    "                'numBins': 30,\n",
+    "                'columnOrExpr': 'dec-0.02'}\n",
+    "        }, \n",
+    "        {\n",
+    "            'dataType': 'fireflyHistogram',\n",
+    "            'tbl_id': 'wiseCatTbl',\n",
+    "            'options': {\n",
+    "                'algorithm': 'fixedSizeBins',\n",
+    "                'fixedBinSizeSelection': 'numBins',\n",
+    "                'numBins': 40,\n",
+    "                'columnOrExpr': 'dec+0.02'   \n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "    result = fc.show_plot(group_id=viewer_id, data=dataH, fireflyData=fireflyDataH )  \n",
+    "    print(result)\n",
+    "    \n",
+    "\n",
+    "# Add fourth chart - 3D plot (plot.ly direct plot)\n",
+    "# in cell 2, 4, 2, 3, \n",
+    "viewer_id =  '3dChartContainer'\n",
+    "r = fc.add_cell(2, 4, 2, 3, 'xyPlots', viewer_id)\n",
+    "\n",
+    "if r['success']:\n",
+    "    data3d = [\n",
+    "        {\n",
+    "            'type': 'scatter3d',\n",
+    "            'name': 'w1-w2',\n",
+    "            'x': \"tables::wiseCatTbl,ra\",\n",
+    "            'y': \"tables::wiseCatTbl,dec\",\n",
+    "            'z': \"tables::wiseCatTbl,w1mpro\",\n",
+    "            'mode' : 'markers',\n",
+    "            'marker' : {\n",
+    "                'size': 8,\n",
+    "                'line': {\n",
+    "                    'color': 'rgb(127, 127, 127, 0.14)',\n",
+    "                    'width': 1}\n",
+    "            },\n",
+    "            'hoverinfo': 'x+y+z'\n",
+    "        }]\n",
+    "\n",
+    "    layout3d = {\n",
+    "        'xaxis': {\n",
+    "            'title': 'ra'\n",
+    "        },\n",
+    "        'yaxis': {\n",
+    "            'title': 'dec'\n",
+    "        },\n",
+    "        'zaxis': {\n",
+    "            'title': 'w1mpro'\n",
+    "        }\n",
+    "    }\n",
```

Comment:
The layout object for 3d should be wrapped into [scene](https://plot.ly/javascript/reference/#layout-scene) - so that the labels are applied to the 3d axes:

```
    layout3d = {
        'scene': {
            'xaxis': {
                'color': 'red',
                'title': 'ra',
                'showline': True
            },
            'yaxis': {
                'color': 'blue',
                'title': 'dec',
                'showline': True
            },
            'zaxis': {
                'color': 'green',
                'title': 'w1mpro',
                'showline': True
            }
        }
    }
```

However, there are still things to fix in firefly, so that scatter3d is not considered to be a scatter plot, and the default layout is not applied to it. I am planning to do it in a separate ticket.